test.js

```diff
@@ -1,5 +1,10 @@
 const hello = 'hello there';
+const another = {value:1}
```

Comment:
'another' is assigned a value but never used. A space is required after '{'. A space is required before '}'. Missing semicolon.